### PR TITLE
Couple of improvements and a feature

### PR DESCRIPTION
**Pull request by jaragunde**
This PR comes with two small improvements (executable flag, protect from empty text) and a feature to add the original author and date of every commit as a text header in them.

I could split this PR into several ones if necessary.

### CODE REVIEW

File: trac2issues.py [hunk 1]
```diff
@@ -322,6 +337,19 @@ def info_has_key(key):
...
             out['state'] = 'closed'
 
+        if self.additional_comments:
+            comment_header = "[Trac import]\n"
+            if info_has_key('reporter'):
+                comment_header += "Reported by: %s\n" % out['creator']['login']
+            try:
+                comment_header += "Original date: %s\n" % \
+                    datetime.fromtimestamp(info['time']/1000000L). \
+                    strftime("%A, %d %B %Y %H:%M")
```
Comment (slinkp) [inline trac2issues.py]: minor style note - I prefer wrapping in parens to trailing backslashes, thanks.

**Comment by slinkp** [comment]
Hi @jaragunde  - thanks for the PR. Sorry I didn't notice the notification earlier.
One minor style comment, then I'm happy to merge this. Thanks!

**Comment by jaragunde** [comment]
Hi @slinkp! No problem, just happy to share :)

I've updated the last comment in the PR to remove the trailing backslashes.

**Comment by slinkp** [comment]
Thanks! Merging - somewhat optimistically, since I no longer have access to any Trac sites and can't easily test this myself :)

**Comment by jaragunde** [comment]
He, that's understandable. I'm about to shut down the trac site I have migrated to GH and when it's done, I'd no longer have a way to test the script either.

You may feel reassured by this: before I sent you the PR, I used the script to migrate these >200 tickets: https://github.com/Igalia/phpreport/issues :)

**Pull request merged**